src/actions/authActions.js

```diff
@@ -0,0 +1,56 @@
+import { postSignUp, postLogIn, getVerify } from "../services/auth-api";
+
+export const SET_USER = 'SET_USER';
+export const SET_USER_ERROR = 'SET_USER_ERROR';
+export const SET_USER_LOADING = 'SET_USER_LOADING';
+
+export const signup = (firstName, lastName, email, password, role) => dispatch => {
```

Comment:
since this function takes so many params, it might be better to pass a user object here instead.

```js
export const signup = user => dispatch => {...}
```

Reply:
we did it!  thanks for the advice!